Resources/Prototypes/Entities/Objects/Materials/materials.yml

```
@@ -587,6 +589,9 @@
   - type: Appearance
   - type: Item
     heldPrefix: bananium
+  - type: Tag
+    tags:
+    - ConstructionMaterial
```

```suggestion
    tags:
    - ConstructionMaterial
    - RawMaterial
```
Test fail real due to this tag being stepped on.